README.md

```diff
@@ -32,41 +32,13 @@ Another example where composition layers are going to shine is displaying text o
+  1. Creating the layers with a given graphics API.
+  1. Positioning the layers and adding them to the session's layers array.
   1. Rendering to the graphics API resources during the frame loop.
 
-### Layer creation
-
-Layers are created by the `XRSession`, using the `requestLayer()` series of fuctions, patterned after the `requestReferenceSpace()` method. Unlike other `request...` methods this function does not return a `Promise`, since it's expected that layer construction will be fairly lightweight. It may return `null`, however, if the `XRSession` does not support the requested layer type. The only layer type that _must_ be supported is `"projection"`. The returned instance depends on the requested type, with each layer type corresponding to it's own interfaces which extends `XRLayer`.
-
-```js
-// Returns an XRProjectionLayer instance
-const projectionLayer = xrSesssion.requestLayer('projection');
-```
-
-This explainer will not cover the details of every possible layer. For additional details please see the separate explainers for each layer type:
-
-  - XRProjectionLayer
-  - [XRQuadLayer](./quad.md)
-  - [XRCylinderLayer](./cylinder.md)
-  - [XREquirectLayer](./equirect.md)
-  - [XRCubeLayer](./cubemap.md)
-
-Layers are not presented to the XR device until they have been added to the `layers` `XRRenderState` property with the `updateRenderState()` method. Setting the `layers` array will override the `baseLayer` if one is present, with `baseLayer` reporting the first element in the `layers` array. Layers will be presented in the order they are given in the `layers` array, with each layers being given in "back-to-front" order. Layers may have alpha blending applied if the layer's `blendSourceAlpha` attribute is `true`, but no depth testing may be performed between layers.
-
-```js
-const projectionLayer = xrSesssion.requestLayer('projection');
-const quadLayer = xrSesssion.requestLayer('quad');
-
-xrSession.updateRenderState({ layers: [projectionLayer, quadLayer] });
-```
-
-`updateRenderState()` _may_ throw an exception if more layers are specified than the `XRSession` supports simultaneously.
-
 ### Graphics API binding
 
-On their own, layers have no visual data, and will be ignored during compositing until provided with a backing `XRLayerSource`. A layer source is typically a GPU resource, like a texture, provided by one of the Web platform's graphics APIs. In order to specify which API is providing the layer sources an `XRGraphicsBinding` instance for the API in question must be created. For example, creating a graphics binding fow WebGL would function like this:
+When a layers is created it is typically backed a GPU resource, like a texture, provided by one of the Web platform's graphics APIs. In order to specify which API is providing the layer's GPU resources an `XRGraphicsBinding` instance for the API in question must be created. For example, creating a graphics binding for WebGL would function like this:
```

Comment:
Well, now I have some issues with this change. How do we specify which type of the resources (framebuffer, texture, texture array, etc) is being used for the layer?

Reply:
We'd have to fix that next

Reply:
In this case the resource type would be dictated by the graphics binding you created the layer with, and each graphics binding would support only a single type. ie: The `XRWebGLGraphicsBinding` would always back layers it creates with `WebGLFramebuffers`. (Which also happens to be why that binding doesn't have a method for creating cube layers, because I don't think you can handle that effectively with a single framebuffer.)

This is a change from what I had previously, where a WebGL2 binding could create both framebuffers or textures, but it seemed like a reasonable simplification. Note that there's nothing here that would prevent you from using two different graphics bindings simultaneously, so you could still mix and match.

Another change that this introduces is that in order to reshape the GPU resources you'd have to recreate the entire layer, but the layer source was always the expensive part of that anyway so I don't think that recreating the layer+source (effectively) here is actually much different. Just means the user would have to copy over the layer positioning attribs.